elevate/views.py

```diff
@@ -60,8 +72,18 @@ def grant_elevated_privileges(self, request, redirect_to):
     def dispatch(self, request):
         redirect_to = request.GET.get(REDIRECT_FIELD_NAME, REDIRECT_URL)
 
+        kwargs = {
+            'url': redirect_to,
+            'host': request.get_host()
+        }
+
+        # NOTE(gabn88): Django 2.1 drops support for the `host` kwarg.
+        if django.VERSION > (2, 0):
+            del kwargs['host']
+            kwargs.update({'allowed_hosts': request.get_host()})
+
```

Comment:
This same chunk of code is repeated twice, I think it would be better to wrap this in our own function. Maybe even a unit test would be in order too.

Reply:
I meant wrapping `is_safe_url`.

Also, could you change the version check to be for 1.11 and above? Django 1.11 will issue a warning if you use the `host` kwarg.